cmake/common/px4_base.cmake

```diff
@@ -486,7 +486,6 @@ function(px4_add_common_flags)
 		-Wextra
 		#-Wshadow # very verbose due to eigen
 		-Wfloat-equal
-		-Wframe-larger-than=1024
```

Comment:
This will take the warning off for the nuttx build for modules that didn't have an explicit max stack size set. Maybe we should add it to nuttx/px4_impl_nuttx.cmake?